fr_FR.yaml

```diff
@@ -1952,25 +1952,25 @@ ui:
       name:
         label: Nom du site
         msg: Le nom ne peut pas être vide.
-        text: "Le nom de ce site, tel qu'il est utilisé dans la balise titre."
+        text: "Le nom de ce site, tel qu'il apparait dans la balise titre."
       site_url:
         label: URL du site
         msg: L'URL ne peut pas être vide.
         validate: Indiquez une URL valide.
         text: L'adresse de ce site.
       short_desc:
-        label: Courte description du site
+        label: Description courte du site
         msg: La description courte ne peut pas être vide.
-        text: "La description courte, telle qu'elle est utilisée dans le tag titre de la page d'accueil."
+        text: "La description courte, telle qu'elle apparait dans le titre de la page d'accueil."
       desc:
         label: Description du site
         msg: La description du site ne peut pas être vide.
-        text: "Décrivez ce site en une phrase, telle qu'elle est utilisée dans la balise meta description."
+        text: "Décrivez ce site en une phrase, telle qu'elle apparait dans la balise meta description."
```

Comment:
Ce « elle » ne fonctionne pas très bien ici pour le « as » général de l’anglais ("Describe this site in one sentence, as used in the meta description tag.").

« Décrivez ce site en une phrase, comme dans la balise *meta description*. »